test/autorest/generated/morecustombaseurigroup/internal/morecustombaseurigroup/paths.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package morecustombaseurigroup
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"net/http"
+	"net/url"
+	"path"
+)
+
+type PathsOperations struct{}
+
+// GetEmptyCreateRequest creates the GetEmpty request.
+func (PathsOperations) GetEmptyCreateRequest(u url.URL, vault string, secret string, keyName string, keyVersion string) (*azcore.Request, error) {
```

Comment:
More unused params.

Reply:
So I believe vault and secret are unused...but keyName and keyVersion should be used I believe. And we're somehow missing SubscriptionID. 
I found this: https://github.com/Azure/autorest.testserver/blob/master/mappings/vanilla/customuri_test12_key1-a0b11188-1f63-4c4f-acd0-729a1f91d6ac.json